### PR TITLE
Add script to deploy launcher

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Script to deploy launcher. Moved from terraform repo.

### How to review 
1. Follow the README and ensure you can deploy launcher using helm.
2. Test as part of https://github.com/ONSdigital/census-eq-concourse/pull/2

### CODE REVIEW

File: scripts/deploy_launcher.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+set -e
+
+if [[ -z "$1" ]]; then
+  echo "RUNNER_URL is mandatory"
+  exit 1
+fi
+
+RUNNER_URL=$1
+IMAGE_TAG="${2:-v1.1.0}"
```
Comment (jonnyshaw89) [inline scripts/deploy_launcher.sh]: Will this become a pain to manage? Everytime we do a new release we have to update the version number?

Comment (MebinAbraham) [reply scripts/deploy_launcher.sh]: It's only a default so doesn't matter too much since you can always set your own tag?

Comment (jonnyshaw89) [reply scripts/deploy_launcher.sh]: yeh i get that, i just wonder how when we will every update the default in this script. It's probably more of an issue for the runner script as there are more frequent releases to that. We will just get lazy and always have an old version deployed locally

Comment (jonnyshaw89) [reply scripts/deploy_launcher.sh]: maybe if the master build also pushed latest then we could default to that?

Comment (MebinAbraham) [reply scripts/deploy_launcher.sh]: That was the alternative, but then got the feeling that we didn't want to push with the tag `latest` but I guess we can push that as well then default it here.

Comment (jonnyshaw89) [reply scripts/deploy_launcher.sh]: yeh we could push with a tag of latest and the commit ref?

Comment (MebinAbraham) [reply scripts/deploy_launcher.sh]: Done

**Review by jonnyshaw89 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**